wazuh/wazuh-kibana-app
# Add agent name to the deployment command

### Description

With this development now the user can add a name to the agent using the deployment wizard of the UI.

This name is defined using the variable `WAZUH_AGENT_NAME`, which is set using the interactive deployment wizard, and added to the installation command.
 
### Issues Resolved

#4573

### Evidence

- When the user has permissions

https://user-images.githubusercontent.com/99441266/197206470-3aa60eea-61e8-47dc-93c6-dcdde54bdb70.mp4

- When the user does not have permissions

https://user-images.githubusercontent.com/99441266/197206745-a3d52f20-4a7c-437b-b864-132df8c549d6.mp4

### Test

1. Go to the agent deployment wizard.
2. Set a name for the agent using the input for this purpose.
3. Check that the variable `WAZUH_AGENT_NAME` has been correctly set and added to the deployment command.

### Check List
